Content/Tools/Editors/PlayerEditor.cs

```diff
@@ -165,6 +166,9 @@ static string GetLocalizedText(string text)
 				return LocalizationHelper.GetText($"Tools.PlayerEditor.{text}");
 			}
 
+			if (BoundingBox.Contains(Main.MouseScreen.ToPoint()))
+				PlayerInput.LockVanillaMouseScroll("DragonLens: Item Editor");

```

Comment:
Reason is incorrect, should be Player editor, not item editor

Reply:
Fixed ✅